lineman/app/components/invitation_form/add_members_modal/add_members_modal.coffee

```diff
@@ -0,0 +1,35 @@
+angular.module('loomioApp').factory 'AddMembersModal', ->
+  templateUrl: 'generated/components/invitation_form/add_members_modal/add_members_modal.html'
+  controller: ($scope, Records, LoadingService, group, AppConfig, FlashService) ->
+    $scope.isDisabled = false
+    $scope.group = group
+    $scope.loading = true
+    $scope.selectedIds = []
+
+    $scope.load = ->
+      Records.memberships.fetchByGroup(group.parent().key, {per: 500})
+
+    $scope.members = ->
+      _.filter group.parent().members(), (user)->
+        !user.isMemberOf(group)
+
+    $scope.canAddMembers = ->
+      $scope.members().length > 0
+
+    LoadingService.applyLoadingFunction($scope, 'load')
+    $scope.load()
+
+    $scope.submit = ->
```

Comment:
I'm not sure how best to use FormService when you're not passing a single model to save. 

Reading over it (FormService) now, I wonder if we want to create a general FormService that can handle any kind of submit, and a more specific ModelFormService that does all the model specific stuff.